src/main/java/com/transferwise/openbanking/client/api/registration/domain/Permission.java

```diff
@@ -0,0 +1,18 @@
+package com.transferwise.openbanking.client.api.registration.domain;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.RequiredArgsConstructor;
+
+@RequiredArgsConstructor
+public enum Permission {
```

Comment:
Maybe better to name it more specific like `RegistrationPermission` etc. Guessing there may be more permissions for different usecases in future.

Reply:
Could be yes, sounds all fine to tweak the name yep.